### PR TITLE
add k8s from pr164

### PR DESCRIPTION
**Pull request by naili-xing**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+stages:
+  - analyze
```
Comment (nudles) [inline .gitlab-ci.yml]: What's the purpose of this CI?

File: dockerfiles/test.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,64 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
```
Comment (nudles) [inline dockerfiles/test.Dockerfile]: for testing only?
what to test ?

Comment (naili-xing) [reply dockerfiles/test.Dockerfile]: test.Dockerfile can be used to create a test container,  we can exec into that container and test the comminication between services or something, it;s not necessary, but we can keep

File: .gitlab-ci.yml (hunk 1 above)
Comment (naili-xing) [reply .gitlab-ci.yml]: this is for deploying the project on the k8s cluster, running all related scripts in this ci job. i didn't use this. i  deploy manually which is easy for debugging each step.

**Pull request merged**